src/_pytest/config/argparsing.py

```diff
@@ -141,19 +140,17 @@ def parse_known_args(
         args: Sequence[Union[str, py.path.local]],
         namespace: Optional[argparse.Namespace] = None,
     ) -> argparse.Namespace:
-        """parses and returns a namespace object with known arguments at this
-        point.
-        """
+        """Parses and returns a namespace object with known arguments at this
+        point."""
```

Comment:
Could/should use imperative mood (also below).
pydocstyle would have caught this.

Reply:
Thanks. So I ran pydocstyle now, and fixes issues that came up: 3b6eedadf33b93b4710b52496d1291f522699ebf

It does seem useful, if we only include some error codes (unfortunately the imperative mood one has too many false positives). I will probably send a PR to add it, but since it also requires fixes to docstrings in testing/ I will leave it to a separate PR.